apps/publications/src/app/components/year-range/year-range.component.html

```diff
@@ -5,14 +5,13 @@
     [matDatepicker]="sdp"
     [max]="startMaxYear"
     [formControl]="startYear"
-    class="disable"
-    readonly />
```

Comment:
I think the readonly property should be kept, the end year date picker still has it.

Reply:
I used to want it like this because the end year shouldn't be empty (there should be at least max value - current year), but for the start year, it makes sense to me, so I wanted to allow the user to delete the start year manually. On the other hand, it looks inconsistently now and the user can clear the filter which deletes the state year, so I agree that `readonly` attribute should probably be here.